### DISCUSSION TITLE
yarn 2+ equivalent for `yarn list` in yarn classic

### DISCUSSION THREAD

**Discussion by GeorgeTaveras1231** [yarnpkg/berry#3784] [Q&A] [answered]
Really surprised this isn't mentioned anywhere in the docs. Personally, I've relied on `yarn list <packageName>` to inspect which versions of a given package are being resolved. How would one do this in `yarn` 2 and 3?

**Comment by arcanis** [comment, marked as answer]
`yarn info -AR lodash`
